src/theme/components/button.ts

```diff
@@ -0,0 +1,53 @@
+import { SystemStyleObject } from '@chakra-ui/theme-tools';
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+import { sizes, typography } from '../shared';
+
+const baseStyle: SystemStyleObject = {
+  borderRadius: 'sm',
+  px: 'lg',
+};
+
+const buttonSizes = {
+  md: {
+    height: sizes.md,
+    ...typography.heading5,
```

Comment:
Maybe it's just me but it feels a bit wired to use the typography for the heading to define the button (I have in mind @GutierrezGo mentioned something on that). Perhaps we can use the one for the body or define one for buttons